### DISCUSSION TITLE
Paging or infinite scroll?

### DISCUSSION THREAD

**Discussion by TimGeyssens** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#65] [Q&A]
what is considered best practise these days?

**Comment by prgriffithsdev** [comment]
I've not used infinite scroll but it sounds interesting but for pagination, I normally make use of https://www.nuget.org/packages/pagedlist.mvc/

**Reply by TimGeyssens** [reply]
Cool thanks for the tip, this one seems to be a maintained fork https://www.nuget.org/packages/X.PagedList

**Reply by prgriffithsdev** [reply]
Yeah, that is the one I think. my bad. :)

**Comment by TimGeyssens** [comment]
Maybe pagination on desktop and on mobile/tablet infinite scroll? With vanilla js... 

Maybe like this (but refactor out jquery) https://dev.to/sakun/a-super-simple-implementation-of-infinite-scrolling-3pnd

**Comment by tcmorris** [comment]
I tend to prefer pagination mainly because it is more deterministic, you know what page you are going to get. The back button should work as expected and you can skip to where you want. 

If you did want infinite scroll then I'd suggest doing it with an optional button, so you can still scroll the page and get to the bottom. If there is limited data then infinite scroll might be ok as is, but you'd probably want to include filters and sort options if there was a lot of data to scroll through. Then you're more in the scenario of a shop front or search listing, which might not be relevant for this project.

**Reply by TimGeyssens** [reply]
good points @tcmorris ! guess v1 we can just do pagination

**Comment by mattbrailsford** [comment]
Pagination. Infinite scrolling is not user friendly at all. You can’t easily link to the page an article is listed on and it’s also trickier to go back to if you go in to view an article and then navigate back.

**Reply by mattbrailsford** [reply]
Maybe look at using a logarithmic pagination technique if you need to plan for lots of pages https://stackoverflow.com/questions/7835752/how-to-do-page-navigation-for-many-many-pages-logarithmic-page-navigation

**Comment by prjseal** [comment]
I think it’s best to do pagination because I was told on multiple occasions that infinite scrolling has accessibility issues

**Comment by Rockerby** [comment]
I like infinite scrolling for fancy things that don’t have ‘pages’ per-se (like social feed type lists) and pages for proper data (like search results or blog listings).

**Comment by TimGeyssens** [comment]
thanks all, pagination it will be :)
